File: src/main/java/ru/arty_bikini/crm/dto/work/TourDTO.java
package ru.arty_bikini.crm.dto.work;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;
import java.time.LocalDateTime;

public class TourDTO {
    private int id;
    private LocalDateTime tour;//доп встреча

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public LocalDateTime getTour() {
        return tour;
    }

    public void setTour(LocalDateTime tour) {
        this.tour = tour;
    }
}

File: common/src/commonMain/kotlin/ru/arty_bikini/crm/dto/work/TourDTO.kt
package ru.arty_bikini.crm.dto.work

import kotlinx.serialization.Serializable
import ru.arty_bikini.crm.dto.EntityDTO

@Serializable
class TourDTO : EntityDTO() {
    override var id = 0
    var tour : Long? = null   //доп встреча
}